### ISSUE TITLE
Documentation: Describe how to manage Reprojection Mode through Application

### ISSUE THREAD

**Issue by srinjoym** [microsoft/MixedRealityToolkit-Unity#7718]
## Describe the problem

Really appreciate the work in #6731 to allow us to set the reprojection mode on Hololens devices using the Camera Profile. I'm looking to extend this feature by toggling the reprojection mode during runtime in one of the applications I'm building. 

So far my approach has been to directly change the `ReprojectionMode` on the instance of `WindowsMixedRealityReprojectionUpdater ` that is initialized on the MainCamera. We need an officially documented way to set the reprojection mode in apps during runtime.

## Describe the solution you'd like
A simple utility to set the mode and accompanying documentation on how to use it.

## Additional context
More context on reprojection: https://docs.microsoft.com/en-us/windows/mixed-reality/hologram-stability

**Note:** The method for setting the reprojection mode in the above document is now outdated.

**Comment by keveleigh** [comment]
>So far my approach has been to directly change the ReprojectionMode on the instance of WindowsMixedRealityReprojectionUpdater that is initialized on the MainCamera. We need an officially documented way to set the reprojection mode in apps during runtime.

That would be my recommendation! The original intent is that the initial setting comes from the camera profile, then any runtime changes would happen directly on the `WindowsMixedRealityReprojectionUpdater` itself.

**Comment by davidkline-ms** [comment]
@srinjoym, If I am reading the request correctly, this can be addressed in documentation. Or is the camera overriding changes made by your app?

**Comment by srinjoym** [comment]
@davidkline-ms Initially, I added the `WindowsMixedRealityReprojectionUpdater ` script to a GameObject in my scene and set the `ReprojectionMode` on that. However, the camera would override the reprojection mode (since it has its own updater).

Directly grabbing the updater from the main camera worked: 
```
var reprojectionUpdater = CameraCache.Main.EnsureComponent<WindowsMixedRealityReprojectionUpdater>();
reprojectionUpdater.ReprojectionMethod = HolographicDepthReprojectionMethod.AutoPlanar;
```

If this is the recommended approach, I think just adding some documentation on how to set the ReprojectionMode during runtime would be great.

**Comment by Holo-Krzysztof** [comment]
I'd appreciate some documentation on the WinRT API side; there's nothing to be found here: https://docs.microsoft.com/en-us/uwp/api/windows.graphics.holographic.holographiccamerarenderingparameters?view=winrt-18362

I only found out this is a thing via a colleague that pointed me to the code and looking inside the HoloLens 2 emulator's `.winmd` files. It'd be nice if there was some sort of official doc outside of MRTK since it's part of a WinRT API.

Also, it's not defined anywhere in the SDK (neither in 18362 nor 19041), so I had to define the COM interface myself to call it from a native application.
It all feels kinda sketchy to me, is this API expected to be stable over time?
